dist/techne.css

```diff
@@ -1,9 +1,9 @@
-@charset "UTF-8";
 /*!
- * Techne v2.0
- * Copyright (c) 2017 SAP SE or an SAP affiliate company.
- * Licensed under Apache License 2.0 (https://github.com/SAP/techne/blob/master/LICENSE)
- */
+* Techne v2.0.0
+* Copyright (c) 2017 SAP SE or an SAP affiliate company.
+* Licensed under Apache License 2.0 (https://github.com/SAP/techne/blob/master/LICENSE)
+*/
+@charset "UTF-8";
```

Comment:
**Note:** Technically the `@charset` needs to be the first thing in the file but you cannot disable this in `gulp-sass`. It is automatically added because of the icon encodings.

A different solution needs to be devised if we want to be 100% fully compliant. To my knowledge, however, this `@charset` declaration is not heavily relied upon by user agents so the impact is probably minimal with this current solution.